docs/en/latest/discovery.md

```diff
@@ -66,7 +67,9 @@ It is very easy for APISIX to extend the discovery client, the basic steps are a
 
 2. Implement the `_M. init_worker()` function for initialization and the `_M. nodes(service_name)` function for obtaining the list of service instance nodes;
 
-3. Convert the registry data into data in APISIX;
+3. If you need the discovery module to export the debugging information online,  implement the `_M. dump_data()` function;
```

Comment:
Style: extra space before `dump_data`.

Reply:
is `_M. dump_data()` -> `_M.dump_data()` ?

Reply:
Yes